client/faucet/README.md

```diff
@@ -0,0 +1,60 @@
+# Faucet
+
+Faucet is a service for creating and funding accounts on Diem Network by treasury compliance account.
+
+It is created for Testnet usage only.
+
+## Mint API
+
+Mint API can create and fund your account. It will fund your account if it is already created onchain.
+
+* Base URL: `http://faucet.testnet.diem.com/`
+* Path: `/mint`
+* Method: POST
+
+URL Query Params:
+
+| param name             | type   | required? |  description                                                         |
+|------------------------|--------|-----------|----------------------------------------------------------------------|
+| `amount`               | int    | Y         | amount of coins to mint                                              |
+| `auth_key`             | string | Y         | your account authentication key                                      |
+| `currency_code`        | string | Y         | the currency code, e.g. XDX                                          |
+| `return_txns`          | bool   | N         | returns the transactions for creating / funding the account          |
+| `is_designated_dealer` | bool   | N         | creates a designated dealer account instead of a parent VASP account |
+
+Notes:
+* By default, the account created is a parent VASP account.
+* Type bool means you set value to a string "true" or "false"
+* The service sbumits 1 transfer funds transactoin for an onchain account `auth_key`.
```

Comment:
```suggestion
* The service submits 1 transfer funds transaction for an onchain account `auth_key`.
```

Reply:
I would change the way this sentence reads:

* For existing accounts as defined by the `auth_key`, the service submits 1 transfer funds transaction
Similarly the following sentence
* For new accounts as defined by the `auth_key`, the service first issues a transaction for creating the account and another for transferring funds.
* All funds transferred come from the account `000000000000000000000000000000dd`
* Clients should retry their request if the requests failed. One reason for failure is that, under load, the service may issue transactions with duplicate sequence numbers. Only one of those transactions will be executed, the rest will fail.